src/index.html

```diff
@@ -13,5 +13,152 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          Surname:
+          <input 
+           type="text" 
+           autocomplete="off">
+        </div>
+         
+        <div class="form_field">
+          Name:
+          <input 
+           type="text"
+           autocomplete="off">
+        </div>
+         
+        <div class="form_field">
+          <label for="age"> How old are you</label>
+          <input 
+           type="number"
+           id="age" 
+           name="age" 
+           min="1" 
+           max="100"
+           value="12"
+           >
+        </div>
+         
+        <div class="form_field">
+          <label> Full date of birth</label>
+          <input 
+              type="date" 
+              name="dateOfbirth">
+        </div>
+
+        <div class="form_field">
+           I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="agreement">
+        </div>
+      </fieldset>
+      
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+
+        <div class="form_field">
+           E-mail:
+          <input type="text"  name="email"  placeholder="email@example.com"  autocomplete="off">
+        </div>
+
+        <div class="form_field">
+           Password:
+          <input 
+            type="password" 
+            name="pass">
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend> An interesting fact about you!</legend>
+
+        <div class="form_field">
+            Do you love cats?
+
+          <label> 
+            <input 
+                type="radio" 
+                name="loveCats" 
+                value="y"> Yes
+          </label>
+
+          <label> 
+            <input 
+                type="radio" 
+                name="loveCats" 
+                value="n"> No
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label> What is your favorite color?</label>
+
+          <input
+             type="color" 
+             name="color">
+        </div>
+
+        <div class="form_field">
+          <label> What time do you go to bed?</label>
+
+          <input
+             type="time" 
+             name="time">
```

Comment:
```suggestion
          <input
             type="time" 
             name="time"
           >
```
fix code style all over the page